### PR TITLE
Fixes #9152: Document node properties usage

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/9152

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
PR rebased

File: 4_advanced_usage/40_node_properties.txt [hunk 1]
```diff
@@ -25,7 +29,7 @@ ${node.properties[datacenter]}
 
 ----
 
-WARNING: Currently node properties can not be used in JavaScript expressions (see following paragraph), since they are evaluated during policy generation and node properties are only made available in the agents. A workaround for this will be provided in a future version of Rudder.
+WARNING: Currently node properties can not be used in JavaScript expressions (see following paragraph), since they are evaluated during policy generation and node properties are only made available in the agents. Starting with Rudder 3.1.14, you can enable a feature swith in "Administration / Setting" to allow node properties expansion in directive parameters. More details can be found at <<_node_properties_expansion_in_directives>>.
```
Comment (fanf) [inline 4_advanced_usage/40_node_properties.txt]: Please also add "and Rudder 3.2.7".

File: 4_advanced_usage/55_node_properties_in_directives.txt [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+----
+
+${node.properties[datacenter][Europe]}
+
+----
+
+Which will be treated as a variable by the agent (replace by its value if defined, kept as is if unknown).
+
+The variable is content is read from /var/rudder/cfengine-community/inputs/properties.d/properties.json.
```
Comment (fanf) [inline 4_advanced_usage/55_node_properties_in_directives.txt]: "You can find more information on node properties can be found in <<_node_properties>>.

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**